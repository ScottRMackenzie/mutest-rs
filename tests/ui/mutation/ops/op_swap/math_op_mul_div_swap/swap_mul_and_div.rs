//@ print-mutants
//@ build
//@ stdout
//@ stderr: empty
//@ mutation-operators: math_op_mul_div_swap

fn f() {
    let _ = 1_u32 as usize * 5_usize;
    let _ = 3 * 6 * 0 * 1 * 0;

    let _ = 99 / 13_usize;
    let _ = 83 / 12 / 9;

    let _ = (131 / 22 * 1) as f64 * 0.1 / 99.0;

    if 0 * 1 > 0 {}

    match 33 / 2 {
        _ if 12 / 1 > 0 => {}
        _ => {}
    }

    for _ in 0..(19 / 3) {}
}

#[test]
fn test() {
    f();
}
