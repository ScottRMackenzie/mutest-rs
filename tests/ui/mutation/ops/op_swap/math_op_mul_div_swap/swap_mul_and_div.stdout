[math_op_mul_div_swap] swap operator `*` for `/` in f at tests/ui/mutation/ops/op_swap/math_op_mul_div_swap/swap_mul_and_div.rs:8:13: 8:37
  <-(0)- test

[math_op_mul_div_swap] swap operator `*` for `/` in f at tests/ui/mutation/ops/op_swap/math_op_mul_div_swap/swap_mul_and_div.rs:9:13: 9:30
  <-(0)- test

[math_op_mul_div_swap] swap operator `*` for `/` in f at tests/ui/mutation/ops/op_swap/math_op_mul_div_swap/swap_mul_and_div.rs:9:13: 9:26
  <-(0)- test

[math_op_mul_div_swap] swap operator `*` for `/` in f at tests/ui/mutation/ops/op_swap/math_op_mul_div_swap/swap_mul_and_div.rs:9:13: 9:22
  <-(0)- test

[math_op_mul_div_swap] swap operator `*` for `/` in f at tests/ui/mutation/ops/op_swap/math_op_mul_div_swap/swap_mul_and_div.rs:9:13: 9:18
  <-(0)- test

[math_op_mul_div_swap] swap operator `/` for `*` in f at tests/ui/mutation/ops/op_swap/math_op_mul_div_swap/swap_mul_and_div.rs:11:13: 11:26
  <-(0)- test

[math_op_mul_div_swap] swap operator `/` for `*` in f at tests/ui/mutation/ops/op_swap/math_op_mul_div_swap/swap_mul_and_div.rs:12:13: 12:24
  <-(0)- test

[math_op_mul_div_swap] swap operator `/` for `*` in f at tests/ui/mutation/ops/op_swap/math_op_mul_div_swap/swap_mul_and_div.rs:12:13: 12:20
  <-(0)- test

[math_op_mul_div_swap] swap operator `/` for `*` in f at tests/ui/mutation/ops/op_swap/math_op_mul_div_swap/swap_mul_and_div.rs:14:13: 14:47
  <-(0)- test

[math_op_mul_div_swap] swap operator `*` for `/` in f at tests/ui/mutation/ops/op_swap/math_op_mul_div_swap/swap_mul_and_div.rs:14:13: 14:40
  <-(0)- test

[math_op_mul_div_swap] swap operator `*` for `/` in f at tests/ui/mutation/ops/op_swap/math_op_mul_div_swap/swap_mul_and_div.rs:14:14: 14:26
  <-(0)- test

[math_op_mul_div_swap] swap operator `/` for `*` in f at tests/ui/mutation/ops/op_swap/math_op_mul_div_swap/swap_mul_and_div.rs:14:14: 14:22
  <-(0)- test

[math_op_mul_div_swap] swap operator `*` for `/` in f at tests/ui/mutation/ops/op_swap/math_op_mul_div_swap/swap_mul_and_div.rs:16:8: 16:13
  <-(0)- test

[math_op_mul_div_swap] swap operator `/` for `*` in f at tests/ui/mutation/ops/op_swap/math_op_mul_div_swap/swap_mul_and_div.rs:18:11: 18:17
  <-(0)- test

[math_op_mul_div_swap] swap operator `/` for `*` in f at tests/ui/mutation/ops/op_swap/math_op_mul_div_swap/swap_mul_and_div.rs:19:14: 19:20
  <-(0)- test

[math_op_mul_div_swap] swap operator `/` for `*` in f at tests/ui/mutation/ops/op_swap/math_op_mul_div_swap/swap_mul_and_div.rs:23:18: 23:24
  <-(0)- test

16 mutations; 16 safe; 0 unsafe (0 tainted)
